docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,267 @@ include::includes/global-attributes.asciidoc[]
+* Click on *Save* and click on *Save* again on the _Configure interfaces and User Access_.
 
-As an example we will use the Management interface as our portal interface. You will need to 'portal' in the setting 'Additional listening daemon(s)'.
+image::docs/images/pfaerohive07.png[scaledwidth="100%",alt="User Profile creation",align=right]
 
-According to our customer scenario, you will associate the correct type the each interfaces.
+'''
 
-    eth0: Management
 
-Make sure that this interface is in an Enabled state for the persistence to occur.
+Create and add the other VLANs ::
+* Click on _Add/Remove_ in the _User Profile_ column.
+* Get back to the beginning of the VLAN creation task.
 
-We also need to set the Default Gateway which will generally be the gateway of the management network.
+'''
 
-Once everything's set, click 'Continue' to proceed with the next step.
 
-Step 3: Database Configuration
-++++++++++++++++++++++++++++++
-This step will configure the MySQL server needed by PacketFence. Database and schema will be created as well as the necessary user for operations. Root account will also be secured if necessary (set a password and disallow remote login).
+Once done with the VLans ::
+* From the _Choose User Profiles_, select the _Default_ tab and Click on you _Registration_ VLAN tag.
+* From the _Authentication_ tab, select the _Isolation_ and the _Production_ VLANs tag. 
+* Click on *Save*
 
-Since Debian based MySQL installations are not "secured", you will need to go through it.
-That step is fairly simple to accomplish and is a one time deal.
+For our example, here what is lookslike, with two SSIDs
 
-In the root account credentials section, enter root as Username and click 'Test'. You'll be prompted for a new root password. Choose a password for the MySQL root user and click 'Save'. You can now enter your newly created password in the root account credentials section and click 'Test'.
+image::docs/images/pfaerohive08.png[scaledwidth="100%",alt="User Profile creation",align=right]
 
-Next section of this step is the PacketFence user account on the MySQL server. Simply leave the default pf username here and choose of a password. This one will automatically be set in the PacketFence configuration where you'll be able to retrieve it in any case. Once the password entered twice, click 'Create user'.
+Then, click on Continue, on top right of the page.
 
-Third section will create the database and load the correct schema on it. Simply leave the default and click 'Create tables and indexes'.
+'''
 
-You should have 'Success' beside these three section, click 'Continue'.
+Push your configuration to your AP::
+* From _Configure and Update Devices_, check your AP in _Device to Update_
+* Click on _Update_
+* Select _update Devices_
+* From _HiveOS Number of devices running earlier versions of HiveOS_, select *Upgrade these devices to the latest version of HiveOS*
```

Comment:
Are you sure you have to upgrade the AP to push its configuration? It looks to me it should be to different things.

Reply:
This is recommended in the deployment guide and it solves some stability troubles.
In the same way, at this point, we have no idea what version the user is running on the AP and if it is in the good version supported by the HiveManager.
So yes, I like to have a homogeneous and coherent system.